config/defaults.go

```diff
@@ -48,6 +51,9 @@ type Defaults struct {
 	// Languages specifies the default build and run container images
 	// for each known language.
 	Languages []LanguageDefault `json:"languages,omitempty"`
+
+	// KillAfter is the duration allowed for pods to respond after timeout.
+	KillAfter float64 `json:"killAfter"`
```

Comment:
We can leave it as "KillAfter" in the code, but perhaps we should consider making it "killAfterSeconds" in the JSON with `json:"killAfterSeconds"`. Without the unit, I don't know what `KillAfter: 20` means.

Reply:
I agree it is bit ambiguous to not able to have a unit, but I think using two different names for the same variable is also a bit confusing. I can add more comments inside the code to improve readability of the code.